introduction/07-faq.asciidoc

```diff
@@ -0,0 +1,15 @@
+== FAQ
+
+To conclude the Introduction segment of the Learning Path, here are some Frequently Asked Questions people have when embarking on their InnerSource journey.
+
+=== What is the cost/overhead?
+
+=== Why not just open source?
+
+=== Will this slow us down?
+
+=== Will we just be reviewing PRs all day every day?
```

Comment:
If so then your core team is understaffed.  A healthy team is staffed so that there is time for assisting contributors _and_ making core contributions yourself.

Reply:
There are also things you can do to mitigate this such as setting expectations, or even SLAs. If contributors expect PR reviews within an hour, maybe you will be stuck reviewing PRs all the time, but if you set an SLA of 1 day or 1 week, this won't be the case.

Reply:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/465/commits/0e4a7d0a88dfae20331af1a86b47e3b1ba80eef2